[0m14:23:19.275326 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f744b267940>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f744bcc9b20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f74498eee80>]}


============================== 14:23:19.285308 | b56b000e-3c84-4b2d-b4cc-259127cbfca6 ==============================
[0m14:23:19.285308 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:23:19.286244 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt init', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:23:19.392967 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object.. Dictionary: {'dir': PosixPath('/root/.dbt')}
[0m14:23:19.393979 [info ] [MainThread]: Creating dbt configuration folder at 
[0m14:23:58.201825 [debug] [MainThread]: Starter project path: /home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/include/starter_project
[0m14:23:58.229875 [info ] [MainThread]: 
Your new dbt project "dermio" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:23:58.230785 [info ] [MainThread]: Setting up your profile.
[0m14:24:01.665938 [error] [MainThread]: Encountered an error:
'type' object is not subscriptable
[0m14:24:01.685103 [error] [MainThread]: Traceback (most recent call last):
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/main.py", line 470, in init
    results = task.run()
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 347, in run
    self.setup_profile(profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 173, in create_profile_from_target
    load_plugin(adapter)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 239, in load_plugin
    return FACTORY.load_plugin(name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 68, in load_plugin
    mod: Any = import_module("." + name, "dbt.adapters")
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/__init__.py", line 16, in <module>
    from dbt.adapters.dremio.connections import DremioConnectionManager
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/connections.py", line 19, in <module>
    from dbt.adapters.dremio.api.cursor import DremioCursor
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/cursor.py", line 20, in <module>
    from dbt.adapters.dremio.api.rest.client import DremioRestClient
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 32, in <module>
    class DremioRestClient:
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 173, in DremioRestClient
    def create_tags(self, dataset_id: str, tags: list[str]):
TypeError: 'type' object is not subscriptable

[0m14:24:01.690128 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 42.599564, "process_user_time": 2.989237, "process_kernel_time": 2.419859, "process_mem_max_rss": "84176", "process_in_blocks": "40800", "process_out_blocks": "80", "command_success": false}
[0m14:24:01.691253 [debug] [MainThread]: Command `dbt init` failed at 14:24:01.690980 after 42.60 seconds
[0m14:24:01.692011 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f744b267940>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7449848160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f74498251c0>]}
[0m14:24:01.692834 [debug] [MainThread]: Flushing usage events
[0m14:24:11.874422 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb80258910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb7ef6db50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb7e8e2070>]}


============================== 14:24:11.882719 | 1fcd8788-187f-4777-a083-9a77ae0f0afa ==============================
[0m14:24:11.882719 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:24:11.910647 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:24:18.173019 [debug] [MainThread]: Starter project path: /home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/include/starter_project
[0m14:24:18.178424 [info ] [MainThread]: 
Your new dbt project "zen" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:24:18.186023 [info ] [MainThread]: Setting up your profile.
[0m14:24:25.007588 [error] [MainThread]: Encountered an error:
'type' object is not subscriptable
[0m14:24:25.178435 [error] [MainThread]: Traceback (most recent call last):
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/main.py", line 470, in init
    results = task.run()
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 347, in run
    self.setup_profile(profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 173, in create_profile_from_target
    load_plugin(adapter)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 239, in load_plugin
    return FACTORY.load_plugin(name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 68, in load_plugin
    mod: Any = import_module("." + name, "dbt.adapters")
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/__init__.py", line 16, in <module>
    from dbt.adapters.dremio.connections import DremioConnectionManager
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/connections.py", line 19, in <module>
    from dbt.adapters.dremio.api.cursor import DremioCursor
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/cursor.py", line 20, in <module>
    from dbt.adapters.dremio.api.rest.client import DremioRestClient
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 32, in <module>
    class DremioRestClient:
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 173, in DremioRestClient
    def create_tags(self, dataset_id: str, tags: list[str]):
TypeError: 'type' object is not subscriptable

[0m14:24:25.181756 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 13.547462, "process_user_time": 2.38485, "process_kernel_time": 0.981997, "process_mem_max_rss": "84276", "process_out_blocks": "64", "command_success": false, "process_in_blocks": "0"}
[0m14:24:25.183434 [debug] [MainThread]: Command `dbt init` failed at 14:24:25.183022 after 13.55 seconds
[0m14:24:25.184756 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb80258910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb7e839130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcb7e8151f0>]}
[0m14:24:25.186015 [debug] [MainThread]: Flushing usage events
[0m16:25:40.904216 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1fb37a160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1fb73c400>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1f9a096a0>]}


============================== 16:25:40.911136 | 76e7e841-919b-444e-a0f4-99664e786ed4 ==============================
[0m16:25:40.911136 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:25:40.912164 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m16:25:50.038509 [debug] [MainThread]: Starter project path: /home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/include/starter_project
[0m16:25:50.045720 [info ] [MainThread]: 
Your new dbt project "zenNetworks" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:25:50.046491 [info ] [MainThread]: Setting up your profile.
[0m16:25:54.362393 [error] [MainThread]: Encountered an error:
'type' object is not subscriptable
[0m16:25:54.373533 [error] [MainThread]: Traceback (most recent call last):
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/cli/main.py", line 470, in init
    results = task.run()
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 347, in run
    self.setup_profile(profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/task/init.py", line 173, in create_profile_from_target
    load_plugin(adapter)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 239, in load_plugin
    return FACTORY.load_plugin(name)
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/factory.py", line 68, in load_plugin
    mod: Any = import_module("." + name, "dbt.adapters")
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/__init__.py", line 16, in <module>
    from dbt.adapters.dremio.connections import DremioConnectionManager
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/connections.py", line 19, in <module>
    from dbt.adapters.dremio.api.cursor import DremioCursor
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/cursor.py", line 20, in <module>
    from dbt.adapters.dremio.api.rest.client import DremioRestClient
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 32, in <module>
    class DremioRestClient:
  File "/home/sara/docker-dbt/venv/lib/python3.8/site-packages/dbt/adapters/dremio/api/rest/client.py", line 173, in DremioRestClient
    def create_tags(self, dataset_id: str, tags: list[str]):
TypeError: 'type' object is not subscriptable

[0m16:25:54.378701 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 13.571559, "process_user_time": 1.532568, "process_kernel_time": 0.347534, "process_mem_max_rss": "84432", "process_in_blocks": "14176", "process_out_blocks": "80", "command_success": false}
[0m16:25:54.379618 [debug] [MainThread]: Command `dbt init` failed at 16:25:54.379448 after 13.57 seconds
[0m16:25:54.380285 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1fb37a160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1f9977130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ff1f98e6460>]}
[0m16:25:54.389285 [debug] [MainThread]: Flushing usage events
[0m16:41:25.606992 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f991aab2d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f99188e99d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f99188e9910>]}


============================== 16:41:25.614121 | 90b64475-b036-4eff-9545-3573a7dcefe1 ==============================
[0m16:41:25.614121 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:41:25.615732 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:41:27.702797 [info ] [MainThread]: A project called zen already exists here.
[0m16:41:27.710810 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 2.207451, "process_user_time": 1.985748, "process_kernel_time": 0.136773, "process_mem_max_rss": "88208", "process_in_blocks": "992", "process_out_blocks": "8"}
[0m16:41:27.711791 [debug] [MainThread]: Command `dbt init` succeeded at 16:41:27.711614 after 2.21 seconds
[0m16:41:27.712518 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f991aab2d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9918984d30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f9918898a90>]}
[0m16:41:27.713187 [debug] [MainThread]: Flushing usage events
[0m16:41:34.541196 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf56000d30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf53e37a30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf53e37970>]}


============================== 16:41:34.548605 | 43b52d65-31ef-4ad3-bb29-dc3cb5ed9128 ==============================
[0m16:41:34.548605 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:41:34.550640 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt init', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:41:37.094113 [debug] [MainThread]: Starter project path: /home/sara/docker-dbt/venv_new/lib/python3.9/site-packages/dbt/include/starter_project
[0m16:41:37.098123 [info ] [MainThread]: 
Your new dbt project "zenn" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:41:37.099420 [info ] [MainThread]: Setting up your profile.
[0m16:45:26.230320 [info ] [MainThread]: Profile zenn written to /root/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:45:26.244313 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 231.7977, "process_user_time": 2.109148, "process_kernel_time": 0.122692, "process_mem_max_rss": "93736", "process_in_blocks": "16", "process_out_blocks": "80"}
[0m16:45:26.247997 [debug] [MainThread]: Command `dbt init` succeeded at 16:45:26.247203 after 231.80 seconds
[0m16:45:26.252360 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf56000d30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf53de6d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faf53e37910>]}
[0m16:45:26.255201 [debug] [MainThread]: Flushing usage events
[0m16:47:33.510532 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d7925fcd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d770989d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d770988e0>]}


============================== 16:47:33.519058 | d91b9c35-955c-4c6d-90d9-20106675bf1c ==============================
[0m16:47:33.519058 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:47:33.520351 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:47:33.594018 [info ] [MainThread]: dbt version: 1.8.7
[0m16:47:33.595175 [info ] [MainThread]: python version: 3.9.22
[0m16:47:33.596792 [info ] [MainThread]: python path: /home/sara/docker-dbt/venv_new/bin/python3.9
[0m16:47:33.598596 [info ] [MainThread]: os info: Linux-5.15.0-131-generic-x86_64-with-glibc2.31
[0m16:47:33.891569 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m16:47:33.972686 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m16:47:33.975444 [info ] [MainThread]: Using dbt_project.yml file at /home/sara/docker-dbt/dbt/dbt_project.yml
[0m16:47:33.977888 [info ] [MainThread]: adapter type: dremio
[0m16:47:33.979646 [info ] [MainThread]: adapter version: 1.8.2
[0m16:47:33.981298 [info ] [MainThread]: Configuration:
[0m16:47:33.982938 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m16:47:33.983934 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m16:47:33.985423 [info ] [MainThread]: Required dependencies:
[0m16:47:33.986758 [debug] [MainThread]: Executing "git --help"
[0m16:47:34.189217 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone             Clone a repository into a new directory\n   init              Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add               Add file contents to the index\n   mv                Move or rename a file, a directory, or a symlink\n   restore           Restore working tree files\n   rm                Remove files from the working tree and from the index\n   sparse-checkout   Initialize and modify the sparse-checkout\n\nexamine the history and state (see also: git help revisions)\n   bisect            Use binary search to find the commit that introduced a bug\n   diff              Show changes between commits, commit and working tree, etc\n   grep              Print lines matching a pattern\n   log               Show commit logs\n   show              Show various types of objects\n   status            Show the working tree status\n\ngrow, mark and tweak your common history\n   branch            List, create, or delete branches\n   commit            Record changes to the repository\n   merge             Join two or more development histories together\n   rebase            Reapply commits on top of another base tip\n   reset             Reset current HEAD to the specified state\n   switch            Switch branches\n   tag               Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch             Download objects and refs from another repository\n   pull              Fetch from and integrate with another repository or a local branch\n   push              Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m16:47:34.194866 [debug] [MainThread]: STDERR: "b''"
[0m16:47:34.202011 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m16:47:34.284876 [info ] [MainThread]: Connection:
[0m16:47:34.287786 [info ] [MainThread]:   cloud_host: None
[0m16:47:34.290762 [info ] [MainThread]:   cloud_project_id: None
[0m16:47:34.292846 [info ] [MainThread]:   software_host: http://localhost:9047
[0m16:47:34.294068 [info ] [MainThread]:   port: 9047
[0m16:47:34.295676 [info ] [MainThread]:   use_ssl: False
[0m16:47:34.296707 [info ] [MainThread]:   environment: None
[0m16:47:34.298933 [info ] [MainThread]:   UID: admin
[0m16:47:34.300177 [info ] [MainThread]:   root_path: no_schema
[0m16:47:34.301648 [info ] [MainThread]:   datalake: $scratch
[0m16:47:34.302506 [info ] [MainThread]:   database: @admin
[0m16:47:34.303189 [info ] [MainThread]:   schema: no_schema
[0m16:47:34.382559 [info ] [MainThread]:   user: admin
[0m16:47:34.799255 [info ] [MainThread]:   username: admin
[0m16:47:34.800550 [info ] [MainThread]:   object_storage_source: $scratch
[0m16:47:34.801431 [info ] [MainThread]:   object_storage_path: no_schema
[0m16:47:34.802100 [info ] [MainThread]:   dremio_space: @admin
[0m16:47:34.803107 [info ] [MainThread]:   dremio_space_folder: no_schema
[0m16:47:35.012758 [info ] [MainThread]: Registered adapter: dremio=1.8.2
[0m16:47:35.521905 [debug] [MainThread]: Acquiring new dremio connection 'debug'
[0m16:47:35.523334 [debug] [MainThread]: dremio adapter: Using dremio connection "debug". fetch=False
[0m16:47:35.524061 [debug] [MainThread]: dremio adapter: On debug: select 1 as id
[0m16:47:35.524800 [debug] [MainThread]: Opening a new connection, currently in state init
[0m16:47:36.096424 [debug] [MainThread]: dremio adapter: Error running SQL: select 1 as id
[0m16:47:36.098166 [debug] [MainThread]: On debug: No close available on handle
[0m16:47:36.099363 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m16:47:36.101044 [info ] [MainThread]: [31m2 checks failed:[0m
[0m16:47:36.104020 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/sara/docker-dbt/dbt/dbt_project.yml> not found

[0m16:47:36.106109 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: /localhost:9047:9047/apiv2/login (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f1d76bce3d0>: Failed to resolve 'http' ([Errno -3] Temporary failure in name resolution)"))

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m16:47:36.109887 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 2.7561421, "process_user_time": 3.007604, "process_kernel_time": 1.261253, "process_mem_max_rss": "95576", "process_in_blocks": "48", "process_out_blocks": "32", "command_success": false}
[0m16:47:36.111373 [debug] [MainThread]: Command `dbt debug` failed at 16:47:36.111044 after 2.76 seconds
[0m16:47:36.112500 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m16:47:36.113637 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d7925fcd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d76bce370>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1d76bce5e0>]}
[0m16:47:36.114960 [debug] [MainThread]: Flushing usage events
[0m11:46:41.291877 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6788f6dc70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6786da5940>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6786da58e0>]}


============================== 11:46:41.404563 | 275c0509-f0b7-42ea-91ff-bda77eecba3c ==============================
[0m11:46:41.404563 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:46:41.407230 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m11:46:44.779330 [debug] [MainThread]: Starter project path: /home/sara/docker-dbt/venv_new/lib/python3.9/site-packages/dbt/include/starter_project
[0m11:46:44.836321 [info ] [MainThread]: 
Your new dbt project "dev" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:46:44.838523 [info ] [MainThread]: Setting up your profile.
[0m11:47:22.571445 [info ] [MainThread]: Profile dev written to /root/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m11:47:22.574901 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 41.466404, "process_user_time": 2.203265, "process_kernel_time": 3.672109, "process_mem_max_rss": "93500", "process_in_blocks": "1104", "process_out_blocks": "88"}
[0m11:47:22.576756 [debug] [MainThread]: Command `dbt init` succeeded at 11:47:22.576319 after 41.47 seconds
[0m11:47:22.578144 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6788f6dc70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6786d82130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f67868e04c0>]}
[0m11:47:22.579649 [debug] [MainThread]: Flushing usage events
[0m11:47:33.639834 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc849b8ac70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc8479c3940>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc8479c38e0>]}


============================== 11:47:33.647273 | 7028d022-c968-4cce-a4b0-f9bbbc39802d ==============================
[0m11:47:33.647273 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:47:33.648863 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:47:33.665029 [info ] [MainThread]: dbt version: 1.8.7
[0m11:47:33.666236 [info ] [MainThread]: python version: 3.9.22
[0m11:47:33.667069 [info ] [MainThread]: python path: /home/sara/docker-dbt/venv_new/bin/python3.9
[0m11:47:33.667819 [info ] [MainThread]: os info: Linux-5.15.0-131-generic-x86_64-with-glibc2.31
[0m11:47:33.848221 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m11:47:33.849608 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m11:47:33.850645 [info ] [MainThread]: Using dbt_project.yml file at /home/sara/docker-dbt/dbt/dbt_project.yml
[0m11:47:33.851790 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m11:47:33.853786 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.3354777, "process_user_time": 3.166915, "process_kernel_time": 0.593796, "process_mem_max_rss": "94236", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m11:47:33.854847 [debug] [MainThread]: Command `dbt debug` failed at 11:47:33.854601 after 0.34 seconds
[0m11:47:33.855739 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc849b8ac70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc8479c3a00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc8479ea2b0>]}
[0m11:47:33.858035 [debug] [MainThread]: Flushing usage events
[0m12:37:54.391541 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba8b373be0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba891aca00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba891ac9a0>]}


============================== 12:37:54.408898 | 87c67e1f-0cf6-418d-8001-08759a8cd575 ==============================
[0m12:37:54.408898 [info ] [MainThread]: Running with dbt=1.8.7
[0m12:37:54.413485 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:37:54.416796 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/sara/docker-dbt/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:37:54.430211 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.1567903, "process_user_time": 2.994335, "process_kernel_time": 0.911319, "process_mem_max_rss": "87340", "process_in_blocks": "67536", "process_out_blocks": "16", "command_success": false}
[0m12:37:54.434105 [debug] [MainThread]: Command `dbt run` failed at 12:37:54.433724 after 0.16 seconds
[0m12:37:54.435457 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba8b373be0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba891acac0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fba891aca60>]}
[0m12:37:54.436366 [debug] [MainThread]: Flushing usage events
[0m12:51:13.341565 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc29be1fd30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc299c58b80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc299c58b20>]}


============================== 12:51:13.397885 | 47436db9-764e-4fb3-bd86-04d77a989012 ==============================
[0m12:51:13.397885 [info ] [MainThread]: Running with dbt=1.8.7
[0m12:51:13.441494 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:51:13.451967 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/sara/docker-dbt/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:51:13.455782 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.34043154, "process_user_time": 3.407747, "process_kernel_time": 0.474496, "process_mem_max_rss": "87152", "process_in_blocks": "720", "process_out_blocks": "8", "command_success": false}
[0m12:51:13.457075 [debug] [MainThread]: Command `dbt run` failed at 12:51:13.456795 after 0.34 seconds
[0m12:51:13.457996 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc29be1fd30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc299c58c40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fc299c58be0>]}
[0m12:51:13.467209 [debug] [MainThread]: Flushing usage events
[0m15:12:51.899540 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f3b441d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f3927aa00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f3927a9a0>]}


============================== 15:12:51.924047 | d3ccdb03-23fd-4a61-b077-59ce6216f53d ==============================
[0m15:12:51.924047 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:12:51.927026 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:12:51.951786 [info ] [MainThread]: dbt version: 1.8.7
[0m15:12:51.956337 [info ] [MainThread]: python version: 3.9.22
[0m15:12:51.973424 [info ] [MainThread]: python path: /home/sara/docker-dbt/venv_new/bin/python3.9
[0m15:12:51.974697 [info ] [MainThread]: os info: Linux-5.15.0-131-generic-x86_64-with-glibc2.31
[0m15:12:52.206400 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m15:12:52.207991 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m15:12:52.209075 [info ] [MainThread]: Using dbt_project.yml file at /home/sara/docker-dbt/dbt/dbt_project.yml
[0m15:12:52.210573 [info ] [MainThread]: adapter type: dremio
[0m15:12:52.211967 [info ] [MainThread]: adapter version: 1.8.2
[0m15:12:52.213416 [info ] [MainThread]: Configuration:
[0m15:12:52.214205 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m15:12:52.214906 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m15:12:52.215572 [info ] [MainThread]: Required dependencies:
[0m15:12:52.216309 [debug] [MainThread]: Executing "git --help"
[0m15:12:52.978177 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone             Clone a repository into a new directory\n   init              Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add               Add file contents to the index\n   mv                Move or rename a file, a directory, or a symlink\n   restore           Restore working tree files\n   rm                Remove files from the working tree and from the index\n   sparse-checkout   Initialize and modify the sparse-checkout\n\nexamine the history and state (see also: git help revisions)\n   bisect            Use binary search to find the commit that introduced a bug\n   diff              Show changes between commits, commit and working tree, etc\n   grep              Print lines matching a pattern\n   log               Show commit logs\n   show              Show various types of objects\n   status            Show the working tree status\n\ngrow, mark and tweak your common history\n   branch            List, create, or delete branches\n   commit            Record changes to the repository\n   merge             Join two or more development histories together\n   rebase            Reapply commits on top of another base tip\n   reset             Reset current HEAD to the specified state\n   switch            Switch branches\n   tag               Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch             Download objects and refs from another repository\n   pull              Fetch from and integrate with another repository or a local branch\n   push              Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m15:12:52.980625 [debug] [MainThread]: STDERR: "b''"
[0m15:12:52.981891 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m15:12:53.064594 [info ] [MainThread]: Connection:
[0m15:12:53.066810 [info ] [MainThread]:   cloud_host: None
[0m15:12:53.067986 [info ] [MainThread]:   cloud_project_id: None
[0m15:12:53.069583 [info ] [MainThread]:   software_host: localhost
[0m15:12:53.071177 [info ] [MainThread]:   port: 9047
[0m15:12:53.075942 [info ] [MainThread]:   use_ssl: False
[0m15:12:53.077856 [info ] [MainThread]:   environment: None
[0m15:12:53.079681 [info ] [MainThread]:   UID: admin
[0m15:12:53.081079 [info ] [MainThread]:   root_path: warehouse
[0m15:12:53.082650 [info ] [MainThread]:   datalake: lakehouse
[0m15:12:53.084108 [info ] [MainThread]:   database: lakehouse
[0m15:12:53.085379 [info ] [MainThread]:   schema: warehouse
[0m15:12:53.086665 [info ] [MainThread]:   user: admin
[0m15:12:53.087951 [info ] [MainThread]:   username: admin
[0m15:12:53.089075 [info ] [MainThread]:   object_storage_source: lakehouse
[0m15:12:53.089918 [info ] [MainThread]:   object_storage_path: warehouse
[0m15:12:53.090689 [info ] [MainThread]:   dremio_space: lakehouse
[0m15:12:53.091530 [info ] [MainThread]:   dremio_space_folder: warehouse
[0m15:12:53.092961 [info ] [MainThread]: Registered adapter: dremio=1.8.2
[0m15:12:53.373930 [debug] [MainThread]: Acquiring new dremio connection 'debug'
[0m15:12:53.375711 [debug] [MainThread]: dremio adapter: Using dremio connection "debug". fetch=False
[0m15:12:53.376683 [debug] [MainThread]: dremio adapter: On debug: select 1 as id
[0m15:12:53.377408 [debug] [MainThread]: Opening a new connection, currently in state init
[0m15:12:53.523556 [debug] [MainThread]: dremio adapter: Error running SQL: select 1 as id
[0m15:12:53.525645 [debug] [MainThread]: On debug: No close available on handle
[0m15:12:53.526794 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m15:12:53.530132 [info ] [MainThread]: [31m2 checks failed:[0m
[0m15:12:53.531512 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/sara/docker-dbt/dbt/dbt_project.yml> not found

[0m15:12:53.533229 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m15:12:53.543122 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 1.7974443, "process_user_time": 2.997283, "process_kernel_time": 0.908267, "process_mem_max_rss": "95136", "process_in_blocks": "120", "process_out_blocks": "16", "command_success": false}
[0m15:12:53.545141 [debug] [MainThread]: Command `dbt debug` failed at 15:12:53.544577 after 1.80 seconds
[0m15:12:53.547058 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m15:12:53.548492 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f3b441d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f38e49fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f6f38e496d0>]}
[0m15:12:53.550081 [debug] [MainThread]: Flushing usage events
[0m10:05:30.585454 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 10:05:30.826561 | 8a6dbe3c-b9ad-463e-b6a7-dbfbb7c430dd ==============================
[0m10:05:30.826561 [info ] [MainThread]: Running with dbt=1.8.7
[0m10:05:30.839687 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/sara/docker-dbt/dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:05:31.253380 [info ] [MainThread]: dbt version: 1.8.7
[0m10:05:31.254635 [info ] [MainThread]: python version: 3.9.22
[0m10:05:31.255564 [info ] [MainThread]: python path: /home/sara/docker-dbt/venv_new/bin/python3.9
[0m10:05:31.256481 [info ] [MainThread]: os info: Linux-5.15.0-131-generic-x86_64-with-glibc2.31
[0m10:05:31.735520 [info ] [MainThread]: Using profiles dir at /home/sara/docker-dbt/dbt
[0m10:05:31.736908 [info ] [MainThread]: Using profiles.yml file at /home/sara/docker-dbt/dbt/profiles.yml
[0m10:05:31.737877 [info ] [MainThread]: Using dbt_project.yml file at /home/sara/docker-dbt/dbt/dbt_project.yml
[0m10:05:31.780456 [info ] [MainThread]: adapter type: dremio
[0m10:05:31.782300 [info ] [MainThread]: adapter version: 1.8.2
[0m10:05:31.783996 [info ] [MainThread]: Configuration:
[0m10:05:31.785858 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m10:05:31.789020 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m10:05:31.790186 [info ] [MainThread]: Required dependencies:
[0m10:05:31.791609 [debug] [MainThread]: Executing "git --help"
[0m10:05:31.957484 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone             Clone a repository into a new directory\n   init              Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add               Add file contents to the index\n   mv                Move or rename a file, a directory, or a symlink\n   restore           Restore working tree files\n   rm                Remove files from the working tree and from the index\n   sparse-checkout   Initialize and modify the sparse-checkout\n\nexamine the history and state (see also: git help revisions)\n   bisect            Use binary search to find the commit that introduced a bug\n   diff              Show changes between commits, commit and working tree, etc\n   grep              Print lines matching a pattern\n   log               Show commit logs\n   show              Show various types of objects\n   status            Show the working tree status\n\ngrow, mark and tweak your common history\n   branch            List, create, or delete branches\n   commit            Record changes to the repository\n   merge             Join two or more development histories together\n   rebase            Reapply commits on top of another base tip\n   reset             Reset current HEAD to the specified state\n   switch            Switch branches\n   tag               Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch             Download objects and refs from another repository\n   pull              Fetch from and integrate with another repository or a local branch\n   push              Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m10:05:31.959108 [debug] [MainThread]: STDERR: "b''"
[0m10:05:31.959952 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m10:05:31.962136 [info ] [MainThread]: Connection:
[0m10:05:31.963540 [info ] [MainThread]:   cloud_host: None
[0m10:05:31.964730 [info ] [MainThread]:   cloud_project_id: None
[0m10:05:31.965978 [info ] [MainThread]:   software_host: localhost
[0m10:05:31.967139 [info ] [MainThread]:   port: 9047
[0m10:05:31.968380 [info ] [MainThread]:   use_ssl: False
[0m10:05:31.969543 [info ] [MainThread]:   environment: None
[0m10:05:31.970662 [info ] [MainThread]:   UID: admin
[0m10:05:31.971787 [info ] [MainThread]:   root_path: warehouse
[0m10:05:31.973038 [info ] [MainThread]:   datalake: lakehouse
[0m10:05:31.974326 [info ] [MainThread]:   database: lakehouse
[0m10:05:31.975525 [info ] [MainThread]:   schema: warehouse
[0m10:05:31.976751 [info ] [MainThread]:   user: admin
[0m10:05:31.978083 [info ] [MainThread]:   username: admin
[0m10:05:31.979472 [info ] [MainThread]:   object_storage_source: lakehouse
[0m10:05:31.980667 [info ] [MainThread]:   object_storage_path: warehouse
[0m10:05:31.981832 [info ] [MainThread]:   dremio_space: lakehouse
[0m10:05:31.982974 [info ] [MainThread]:   dremio_space_folder: warehouse
[0m10:05:31.984881 [info ] [MainThread]: Registered adapter: dremio=1.8.2
[0m10:05:32.282940 [debug] [MainThread]: Acquiring new dremio connection 'debug'
[0m10:05:32.284343 [debug] [MainThread]: dremio adapter: Using dremio connection "debug". fetch=False
[0m10:05:32.285177 [debug] [MainThread]: dremio adapter: On debug: select 1 as id
[0m10:05:32.286002 [debug] [MainThread]: Opening a new connection, currently in state init
[0m10:05:33.798273 [debug] [MainThread]: dremio adapter: Connected to db: lakehouse
[0m10:05:36.400459 [debug] [MainThread]: dremio adapter: Job State = PLANNING
[0m10:05:36.766361 [debug] [MainThread]: dremio adapter: Job State = QUEUED
[0m10:05:38.947044 [debug] [MainThread]: dremio adapter: Job State = EXECUTION_PLANNING
[0m10:05:39.229771 [debug] [MainThread]: dremio adapter: Job State = RUNNING
[0m10:05:41.197161 [debug] [MainThread]: dremio adapter: Job State = COMPLETED
[0m10:05:41.198474 [debug] [MainThread]: dremio adapter: SQL status: 1 in 8.91 seconds
[0m10:05:41.201059 [debug] [MainThread]: On debug: Close
[0m10:05:41.202055 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m10:05:41.203350 [info ] [MainThread]: [31m1 check failed:[0m
[0m10:05:41.204346 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/sara/docker-dbt/dbt/dbt_project.yml> not found

[0m10:05:41.210538 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 10.813286, "process_user_time": 3.111414, "process_kernel_time": 0.992045, "process_mem_max_rss": "95256", "process_in_blocks": "34848", "process_out_blocks": "16", "command_success": false}
[0m10:05:41.212378 [debug] [MainThread]: Command `dbt debug` failed at 10:05:41.212021 after 10.82 seconds
[0m10:05:41.213848 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m10:05:41.215133 [debug] [MainThread]: Flushing usage events
